Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -698,16 +698,45 @@ static GridArea insertIntoGrid(Grid& grid, RenderBox& child, const GridArea& are
     return clamped;
 }
 
-static inline bool isMasonryRows(const RenderStyle& style)
+bool RenderGrid::isMasonryRows() const
 {
-    return style.gridMasonryRows();
+    return style().gridMasonryRows();
 }
 
-static inline bool isMasonryColumns(const RenderStyle& style)
+// Masonry Spec Section 2
+// "If masonry is specified for both grid-template-columns and grid-template-rows, then the used value for grid-template-columns is none,
+// and thus the inline axis will be the grid axis."
+
+bool RenderGrid::isMasonryColumns() const
 {
-    return !isMasonryRows(style) && style.gridMasonryColumns();
+    return !isMasonryRows() && style().gridMasonryColumns();
+}
+
+// Masonry Spec Section 2.3.1 repeat(auto-fit)
+// "repeat(auto-fit) behaves as repeat(auto-fill) when the other axis is a masonry axis."
+// We need to lie here that we are are really an auto-fill instead of an auto-fit.
+
```

Comment:
Ditto: We don't use this whitespace between method comment and method.

Reply:
Fixed